tracer/test/test-applications/debugger/Samples.Probes/Properties/launchSettings.json

```diff
@@ -2,20 +2,20 @@
     "profiles": {
       "Samples.Probes": {
         "commandName": "Project",
-        "commandLineArgs": "--test-name Samples.Probes.SmokeTests.HasFieldLazyValueInitialized --listen-url http://127.0.0.1:1234/",
+        "commandLineArgs": "--test-name Samples.Probes.SmokeTests.StaticType",
         "environmentVariables": {
-          "DD_LOG_LEVEL": "DEBUG",
-        "CORECLR_PROFILER_PATH": "$(SolutionDir)shared\\bin\\monitoring-home\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
-        "COR_PROFILER_PATH": "$(SolutionDir)shared\\bin\\monitoring-home\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
-          "COR_ENABLE_PROFILING": "1",
-          "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
-          "CORECLR_ENABLE_PROFILING": "1",
-          "DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1",
-          "DD_DOTNET_TRACER_HOME": "$(SolutionDir)shared\\bin\\monitoring-home\\",
-          "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
-          "DD_DUMP_ILREWRITE_ENABLED": "1",
-          "DD_WRITE_INSTRUMENTATION_TO_DISK": "1",
-          "DD_SERVICE": "debugger_demo_dotnet"
+          //"DD_LOG_LEVEL": "DEBUG",
+          //"CORECLR_PROFILER_PATH": "$(SolutionDir)shared\\bin\\monitoring-home\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
+          //"COR_PROFILER_PATH": "$(SolutionDir)shared\\bin\\monitoring-home\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
+          //"COR_ENABLE_PROFILING": "1",
+          //"COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+          //"CORECLR_ENABLE_PROFILING": "1",
+          //"DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1",
+          //"DD_DOTNET_TRACER_HOME": "$(SolutionDir)shared\\bin\\monitoring-home\\",
+          //"CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+          //"DD_DUMP_ILREWRITE_ENABLED": "1",
+          //"DD_WRITE_INSTRUMENTATION_TO_DISK": "1",
+          //"DD_SERVICE": "debugger_demo_dotnet"
```

Comment:
Should these still be commented out?

Reply:
No 😅 
Uncommented here 6871c8e8c072d551bf56c7225f59bb350f50396e